LDAP-Auth/Config/PluginConfiguration.cs

```diff
@@ -98,6 +101,21 @@ public PluginConfiguration()
         /// </summary>
         public bool EnableCaseInsensitiveUsername { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ldap client cert path.
+        /// </summary>
+        public string LdapClientCert { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ldap client cert path.
+        /// </summary>
+        public string LdapClientKey { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ldap root CA path.
+        /// </summary>
+        public string LdapRootCa { get; set; }
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets the ldap client cert path.
        /// </summary>
        public string LdapClientCertPath { get; set; }

        /// <summary>
        /// Gets or sets the ldap client cert path.
        /// </summary>
        public string LdapClientKeyPath { get; set; }

        /// <summary>
        /// Gets or sets the ldap root CA path.
        /// </summary>
        public string LdapRootCaPath { get; set; }
```

Make the properties descriptive of what they contain